extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -319,6 +331,28 @@ export class AmpSelector extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Handles selectUp events.
+   * @param {number} delta
+   */
+  select_(delta) {
+    // Change the selection to the next element in the specified direction.
+    // The selection should loop around if the user attempts to go one
+    // past the beginning or end.
+    const previousIndex = this.options_.indexOf(this.selectedOptions_[0]);
+    const index = previousIndex + delta;
+    const normalizedIndex = index > 0 ?
+      index % this.options_.length :
+      ((index % this.options_.length) +
+      this.options_.length) % this.options_.length;
+
+    user().assert(normalizedIndex >= 0,
```

Comment:
With the true modulo change, is it mathematically possible that `normalizedIndex` could ever be negative? I'm not sure this user assert is necessary.